File: fix_fatima_fr.js
const fs = require('fs');

// Read the file
const content = fs.readFileSync('c:/Users/RC71/Documents/etranfers.pt/script.js', 'utf8');

// Replace the French Fátima description  
const oldText = '"L'un des plus grands centres de pèlerinage marial au monde !"';
const newText = '"L\'un des plus grands centres de pèlerinage marial au monde, le Sanctuaire de Fátima reçoit des millions de visiteurs chaque année, étant aujourd\'hui un lieu de foi, de réflexion et de profonde dévotion."';

const updatedContent = content.replace(oldText, newText);

// Write back
fs.writeFileSync('c:/Users/RC71/Documents/etranfers.pt/script.js', updatedContent, 'utf8');

console.log('French Fátima translation updated successfully!');

File: fix_fatima_fr.py
import re

# Read the file
with open('c:/Users/RC71/Documents/etranfers.pt/script.js', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace the French Fátima description
old_text = '"L'un des plus grands centres de pèlerinage marial au monde !"'
new_text = '"L\'un des plus grands centres de pèlerinage marial au monde, le Sanctuaire de Fátima reçoit des millions de visiteurs chaque année, étant aujourd\'hui un lieu de foi, de réflexion et de profonde dévotion."'

content = content.replace(old_text, new_text)

# Write back
with open('c:/Users/RC71/Documents/etranfers.pt/script.js', 'w', encoding='utf-8') as f:
    f.write(content)

print("French Fátima translation updated successfully!")
